k8s/base/env/jicofo.env

```diff
@@ -65,4 +65,4 @@ MAX_BRIDGE_PARTICIPANTS=-1
 #
 # Additionally, you can use the fully qualified class name for custom
 # BridgeSelectionStrategy implementations.
-OCTO_BRIDGE_SELECTION_STRATEGY=SplitBridgeSelectionStrategy
+OCTO_BRIDGE_SELECTION_STRATEGY=SingleBridgeSelectionStrategy
```

Comment:
You should also explain please why you made this change in the commit description. What is the reason to make this change. Thanks